src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.props

```diff
@@ -82,4 +82,16 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(DisableImplicitNamespaceImports_DotNet)' != 'true'
+                      and '$(TargetFrameworkIdentifier)' == '.NETCoreApp'
+                      and $([MSBuild]::VersionGreaterThanOrEquals($(_TargetFrameworkVersionWithoutV), '6.0'))">
+    <Import Include="System" />
+    <Import Include="System.Collections.Generic" />
+    <Import Include="System.IO" />
+    <Import Include="System.Linq" />
+    <Import Include="System.Net.Http" />
```

Comment:
I still maintain `System.Net.Http` shouldn't be in the default set, as it is generally irrelevant in the desktop space.

Reply:
I thought we'd agreed to remove it too. @JunTaoLuo?

Reply:
"Irrelevant" is a bit strong, a lot of desktop/mobile apps use `HttpClient`. That said, their use is probably confined to a few files and may not justify a global namespace.

Reply:
I think Http should be in the default set.

Reply:
We discussed this several times already so I'm going to include it for now. The motivation is that we think this is used often enough to warrant being included by default in the base SDK. We had support from Immo on this as well.

Unless this breaks certain scenarios (e.g. type name conflict) I'm not going to remove it here. If you feel this needs more discussion @RussKie, please file an issue to track that discussion.